### DISCUSSION TITLE
Add a button to fast export/download detections

### DISCUSSION THREAD

**Discussion by izzygold** [blakeblackshear/frigate#14312] [Ideas]
Hi, 
Will it be possible to add a button that will do "fast export" for detections?
For example, I am looking through my reviews, sees something interesting that I want to share - I'd like to share it with a click of a button.

**Comment by hawkeye217** [comment]
A share button has been added for review items in the next version of Frigate.

In the current version, you can right click (desktop) or long press (mobile) on a review item and choose the export button at the top for an immediate export of that review item.

**Reply by izzygold** [reply]
oh I see this now, I was expected a pop up on right click. I now see the the top bar changes. thanks!

**Reply by izzygold** [reply]
Can you add support to select multiple item export?

**Reply by hawkeye217** [reply]
That's a bit more complicated. There's already an open feature requeset for it. https://github.com/blakeblackshear/frigate/issues/12603
